tests/integration/test_issue_2001.py

```diff
@@ -0,0 +1,37 @@
+import os.path
+import subprocess
+import sys
+
+from pex.testing import PY38, ensure_python_interpreter, make_env, run_pex_command
+from pex.typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from typing import Any
+
+
+def test_compile_error_as_warning(
+    tmpdir,  # type: Any
+):
+    # type: (...) -> None
+    pex = os.path.join(str(tmpdir), "pex.pex")
+    run_pex_command(
+        args=[
+            "--include-tools",
+            "aenum",
```

Comment:
Pleas pin to the current latest release with "aenum==3.1.11" so that this test doesn't break when they publish new releases.